From 62952dfbf4a719e41aedd62b018ed57438bc4f9d Mon Sep 17 00:00:00 2001
From: Nikhil Badola <nikhil.badola@freescale.com>
Date: Tue, 24 Dec 2013 12:25:09 +0530
Subject: [PATCH 0804/1089] powerpc/t2080: Change t2080 USB controller version

[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]

Change USB controller version to 2.5 in compatible string for T2080

Signed-off-by: Nikhil Badola <nikhil.badola@freescale.com>
Change-Id: I4a539e3e0984e418c09a3d64405f9b844f404289
Reviewed-on: http://git.am.freescale.net:8181/7459
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Jose Rivera <German.Rivera@freescale.com>
(cherry picked from commit cd37bcac41666cccb7fdf84e475967630848d257)
Change-Id: I4a539e3e0984e418c09a3d64405f9b844f404289
Reviewed-on: http://git.am.freescale.net:8181/9835
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Jose Rivera <German.Rivera@freescale.com>
---
 arch/powerpc/boot/dts/fsl/t2081si-post.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/powerpc/boot/dts/fsl/t2081si-post.dtsi b/arch/powerpc/boot/dts/fsl/t2081si-post.dtsi
index cc5b471..1fb0607 100644
--- a/arch/powerpc/boot/dts/fsl/t2081si-post.dtsi
+++ b/arch/powerpc/boot/dts/fsl/t2081si-post.dtsi
@@ -634,13 +634,13 @@
 	};
 /include/ "qoriq-usb2-mph-0.dtsi"
 	usb0: usb@210000 {
-		compatible = "fsl-usb2-mph-v2.4", "fsl-usb2-mph";
+		compatible = "fsl-usb2-mph-v2.5", "fsl-usb2-mph";
 		phy_type = "utmi";
 		port0;
 	};
 /include/ "qoriq-usb2-dr-0.dtsi"
 	usb1: usb@211000 {
-		compatible = "fsl-usb2-dr-v2.4", "fsl-usb2-dr";
+		compatible = "fsl-usb2-dr-v2.5", "fsl-usb2-dr";
 		dr_mode = "host";
 		phy_type = "utmi";
 	};
-- 
2.0.2

